SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+
+* **Removal**: A board member may be removed by a two-thirds majority vote of the full board, if before any meeting of the board at which a vote on removal will be made the member in question is given electronic or written notification of the board’s intention to discuss their case and is given the opportunity to be heard at a meeting of the board.
+Alternatively a member of the board can be removed by a vote of the Voting Members, following the same procedure laid out below for the removal of a member of the Steering Committee.
+
+* **Meetings**:  The board shall have a meeting at least every quarter at times and places fixed by the board.
+Board meetings shall provide at least one week (7 days) notice to all board members and the Steering Committee.
+Notice of meetings shall specify the place, day, and hour of meeting.
+The purpose of the meeting needs not be specified.
+
+* **Board officers**: The officers of the board shall be a board chair, vice-chair, and secretary, all of whom shall be chosen by, and serve at the pleasure of, the board.
+Officers shall serve one year terms (as officers) and can serve consecutive terms.
+
+* **Board Chair**: The board chair shall lead the board in performing its duties and responsibilities, including, if present, leading all meetings of the board.
+
+* **Vice-chair**: In the absence of the board chair, the vice-chair shall perform the duties of the board chair.
+When so acting, the vice-chair shall have all the powers of and be subject to all the restrictions upon the board chair.
+
+* **Secretary**: The secretary shall be responsible for keeping the minutes of all meetings and actions of the board.
+The minutes of each meeting shall state the time and place that it was held, the members present, and such other information as shall be necessary to determine the actions taken.
+The minutes of all board meetings shall be made publicly accessible to the whole SunPy community.
+
+* **Advise and Consent**: The steering committee shall present the state of the project and future plans at meetings of the board.
+The advisory board shall advise and consent on those plans.
+
+
+## The Steering Committee
+The role of the steering committee, which is headed by the Lead Developer, is to manage the day to day operations of the SunPy project.
+The steering committee shall be responsible for the running of the project under advisement of the board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus
+* delegating responsibility and authority to other community roles
+* planning and managing finances and financial processes of the project.
+
+### Composition
+
+The steering committee is formed of three roles:
```

Comment:
I would expect a steering committee consist of three roles with equal power such that they can be checks on each other. Feels like the lead developer is the real power and the other two especially the deputy fall below the lead developer in the org chart.

Reply:
This is is a wording and clarity issue.

My intention here is that all the people on the steering committee are equals, just with defined roles (which was a suggestion at the coordination meeting). So the Lead developer chairs the committee (is responsible for organising meetings etc), the deputy deputises and the financial officer is primarily responsible for financial reports etc.

The power to make any decisions lies with the committee as a whole, and as stated below if a decision can not be reached by consensus then a majority is required.

Reply:
I have hopefully clarified this below?